react/features/invite/components/add-people-dialog/web/InviteContactsForm.js

```diff
@@ -0,0 +1,489 @@
+                        elemBefore: tagAvatar
+                    },
+                    value: phone
+                });
+            }
+        });
+
+        const numbers = response.filter(item => item.type === 'phone');
+        const telephoneIcon = this._renderTelephoneIcon();
+
+        const numberDisplayItems = numbers.map(number => {
+            const numberNotAllowedMessage
+                = number.allowed ? '' : t('addPeople.countryNotSupported');
+            const countryCodeReminder = number.showCountryCodeReminder
+                ? t('addPeople.countryReminder') : '';
+            const description
+                = `${numberNotAllowedMessage} ${countryCodeReminder}`.trim();
+
+            return {
+                filterValues: [
+                    number.originalEntry,
+                    number.number
+                ],
+                content: t('addPeople.telephone', { number: number.number }),
+                description,
+                isDisabled: !number.allowed,
+                elemBefore: telephoneIcon,
+                item: number,
+                tag: {
+                    elemBefore: telephoneIcon
+                },
+                value: number.number
+            };
+        });
+
+        return [
+            ...userDisplayItems,
+            ...numberDisplayItems
+        ];
+    }
+
+    _query: (string) => Promise<Array<Object>>;
+
+    _onClearItems: () => void
+
+    /**
+     * Clears the selected items from state and form.
+     *
+     * @returns {void}
+     */
+    _onClearItems() {
+        if (this._multiselect) {
+            this._multiselect.setSelectedItems([]);
+        }
+        this.setState({ inviteItems: [] });
+    }
+
+    /**
+     * Renders the add/cancel actions for the form.
+     *
+     * @returns {ReactElement|null}
+     */
+    _renderFormActions() {
+        const { inviteItems } = this.state;
+        const { t } = this.props;
+
+        if (!inviteItems.length) {
+            return null;
+        }
+
+        return (
+            <div className = 'invite-more-dialog invite-buttons'>
+                <a
+                    className = 'invite-more-dialog invite-buttons-cancel'
+                    onClick = { this._onClearItems }>
+                    {t('dialog.Cancel')}
+                </a>
+                <a
+                    className = 'invite-more-dialog invite-buttons-add'
+                    onClick = { this._onSubmit }>
+                    {t('addPeople.add')}
+                </a>
+            </div>
+        );
+    }
+
+    /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @private
+     * @returns {ReactElement|null}
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
+        }
+
+        const { t } = this.props;
+        const supportString = t('inlineDialogFailure.supportMsg');
+        const supportLink = interfaceConfig.SUPPORT_URL;
```

Comment:
check if it's undefined?

Reply:
Would that be needed? The support url is hardcoded in interface_config.js atm. If it could indeed be undefined, what should we default to?

Reply:
Maybe nothing? Just ddon't show anything.

Reply:
Maybe it shouldn't have a default value, since people running their own installs would have a link to us. Anyway, food for thought.